lib/manager/maven/index.ts

```diff
@@ -7,6 +7,10 @@ export { updateDependency } from './update';
 export const language = ProgrammingLanguage.Java;
 
 export const defaultConfig = {
-  fileMatch: ['\\.pom\\.xml$', '(^|/)pom\\.xml$'],
+  fileMatch: [
+    '\\.pom\\.xml$',
+    '(^|/)pom\\.xml$',
```

Comment:
Is it worth unifying these two while we're making changes?

Reply:
We can do that. This should be `'(((^|/)pom)|(\\.pom))\\.xml$',`, correct?

Reply:
Maybe just `'(^|/|\\.)pom\\.xml$'` ?

Reply:
yeah, looks better 👍